src/views/ScorePage/ScorePage.js

```diff
@@ -0,0 +1,89 @@
+import './ScorePage.css';
+import { scoreLocalStorage, saveHighScore } from '../../score/user-score';
+import Button from '../../components/Button';
+import '../../components/Button.css';
+
+export function RenderScorePage() {
+  const highScores = JSON.parse(localStorage.getItem('highScores')) || [];
+  const recentUserScore = 12;
+  const questionsNumber = 12;
+
+  const div = document.createElement('div');
+  div.setAttribute('id', 'scorePage');
+  div.append(
+    renderGraphics(),
+    RenderScore(recentUserScore, questionsNumber),
+    renderNickForm(),
+    renderMenuButton(),
+    renderPlayAgainButton(),
+  );
+  document.querySelector('#app').append(div);
```

Comment:
div variable name can be confusing (how about two divs in one function :P ) - it is better to use a 'container' instead